src/mds/topology/topology.cpp

```diff
@@ -75,19 +92,41 @@ int TopologyImpl::AddLogicalPool(const LogicalPool &data) {
     }
 }
 
-int TopologyImpl::AddPhysicalPool(const PhysicalPool &data) {
+int TopologyImpl::AddPhysicalPoolJustForTest(const PhysicalPool &data) {
+    ReadLockGuard rlockPoolset(poolsetMutex_);
     WriteLockGuard wlockPhysicalPool(physicalPoolMutex_);
     if (physicalPoolMap_.find(data.GetId()) == physicalPoolMap_.end()) {
         if (!storage_->StoragePhysicalPool(data)) {
             return kTopoErrCodeStorgeFail;
         }
+        // it->second.AddPhysicalPool(data.GetId());
```

Comment:
Check everything and delete useless code.

Reply:
> Check everything and delete useless code.

fix